modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -257,11 +261,25 @@ private Set<AssetVocabulary> _getNoninternalAssetVocabularies(
 	private List<AssetVocabulary> _getNoninternalAssetVocabularies(
 		String itemClassName, long itemClassTypeId, long scopeGroupId) {
 
+		long[] groupsIds;
+
+		try {
```

Comment:
Why didn't you use the com.liferay.depot.group.provider.SiteConnectedGroupGroupProvider?? 

It does the same

Reply:
Thanks Roberto, I forgot that was an option. I've made that update.